moduls/fastq_tools.py

```diff
@@ -0,0 +1,65 @@
+def GC_cont_check(sequence_for_filtering: str , gc_bounds: list) -> bool:
+    """
+    Check the GC content of the sequence for filtering
+    :param sequence_for_filtering: analyzed sequence
+    :param gc_bounds: list with limitation for GC content
+    :return: bool argument for filtering in fasta_filtering function
+    """
+    GC_counter = 0
+    for i in sequence_for_filtering:
+        if i == "C" or i == "G" or i == "c" or i == "g":
+            GC_counter += 1
+    GC_calculc = GC_counter/len(sequence_for_filtering)*100
+    if len(gc_bounds)==1:
+        if GC_calculc <= gc_bounds[0]:
+            return True
+    if len(gc_bounds) == 2:
+        if GC_calculc <= gc_bounds[1] and GC_calculc >= gc_bounds[0]:
+            return True
```

Comment:
Е-еей, ну  тут совсем все намешано. 
1) Ты получается разделил 2 ветви логики таким образом, что дважды делаешь по сути одно и то же. Можно было бы свети один случай к другому
2) Не надо if ... return True, сразу возвращаем результат булевой операции
3) Можно чуть сократить записать сравнения с нижней и верхней границей
4) Если подано число, то у него нет длины и все падает с ошибкой. Надо тут проверять типы
```suggestion
    if isinstance(gc_bounds, int ) or isinstance(gc_bounds, float):
        gc_bounds = (0, gc_bounds)
    return gc_bounds[0] <= GC_calculc<= gc_bounds[0]
```